Firmware/xyzcal.cpp

```diff
@@ -135,9 +135,9 @@ pos_mm_t pos_2_mm(float pos){
 	return pos * 0.01f;
 }
 
-void xyzcal_meassure_center(void)
+void xyzcal_meassure_enter(void)
 {
-	DBG(_n("xyzcal_meassure_center\n"));
+	DBG(_n("xyzcal_meassure_enter\n"));
```

Comment:
Is it meassure or measure?

Reply:
It's measure. Isn't it fixed in the next commit?

Reply:
This misspelling bothered me for a looong time ;)